src/Factory/RestControllerFactory.php

```diff
@@ -237,6 +237,12 @@ protected function setControllerOptions(array $config, RestController $controlle
 
                         $query  = $request->getQuery();
                         $params = new Parameters([]);
+
+                        // If a query Input Filter exists, use its keys as query whitelist
+                        // instead of admin query whitelist field
```

Comment:
I'd argue we should _merge_ those from configuration with those from the input filter.

Reply:
I agree with that, I'll update the code